### PR TITLE
Add OData QuerySegment documentation

### PR DESCRIPTION
**Pull request by osoykan**

### CODE REVIEW

File: content/refguide/odata-query-options.md [hunk 1]
```diff
@@ -141,3 +141,18 @@ You can compare values against the `null` literal. For example: `?$filter=Name e
...
+POST: `/Products/$query` 
+Content-Type : `text-plain`
+
+And the remaining part can be placed in the body of the request. The important point here is that the request **MUST** be **POST**, and `Content-Type` **MUST** be `text-plain` for this operation. Another example is; you can pass some part of the query to the request, in this case, our query looks like this; 
+
+POST: `/Products/$query?$filter=Age gt 65` 
+Content-Type: `text-plain`
+Request Body: `$select=Name,Price`
+
+Please note that; since the **Request Body** is `text-plain`, it **MUST** adhere to the _URL encoding_ principles. Meaning, no spaces, tabs, or newlines are allowed.
```
Comment (Adam-Dupaski) [inline content/refguide/odata-query-options.md]: @osoykan this should be "since the **Content_Type** is `text-plain`" , correct?

Comment (osoykan) [reply content/refguide/odata-query-options.md]: Yes, correct.

**Comment by osoykan** [comment]
We will make these changes from another MR under @joostverhoog's account.

**Comment by Adam-Dupaski** [comment]
@osoykan I've already made a commit in review, why is that necessary?

**Comment by osoykan** [comment]
@joostverhoog wanted to make some changes/additions, so I targeted his branch now.

**Pull request closed without merging**